docs/Users_Guide/mode-td.rst

```diff
@@ -197,7 +197,7 @@ The usage statement for the MODE-TD tool is listed below: The command line switc
   Usage: mtd
          -fcst    file_1 ... file_n | file_list
          -obs     file_1 ... file_n | file_list
-         -single  file_1 ... file_n | file_list
+         [-single  file_1 ... file_n | file_list]
```

Comment:
Why are brackets needed here but not for `fcst` or `obs`?

Reply:
The brackets are used to denote *optional* flags, while no brackets indicates required. There's just too much info to convey here. You can either run with `-fcst` AND `-obs` ... or `-single` by itself.

This is the same situation as series_analysis where you can use `-fcst` AND `-obs`... or `-both` by itself. And in series_analysis we put `[-both]` in brackets:
```
Usage: series_analysis
	-fcst  file_1 ... file_n | file_list
	-obs   file_1 ... file_n | file_list
	[-both file_1 ... file_n | file_list]
```
So right or wrong, I was making it more consistent.

Reply:
Sounds good to me, it was mostly my curiosity. I like the consistency with series_analysis. Thanks!